OMCompiler/Compiler/runtime/CMakeLists.txt

```diff
@@ -279,7 +279,7 @@ else()
   set(RT_LDFLAGS_GENERATED_CODE " -lOpenModelicaRuntimeC -lomcmemory -llapack -lblas -lm -lpthread -rdynamic")
   set(RT_LDFLAGS_GENERATED_CODE_SIM " -lSimulationRuntimeC -lOptimizationRuntime -lOpenModelicaRuntimeC -lomcmemory -lzlib -llapack -lblas -lm -ldl -lpthread -lgfortran -lstdc++ -rdynamic -Wl,--no-undefined")
   set(RT_LDFLAGS_GENERATED_CODE_SOURCE_FMU " -lOpenModelicaFMIRuntimeC -llapack -lblas -lm -lpthread -rdynamic -Wl,--no-undefined")
-  set(RT_LDFLAGS_GENERATED_CODE_SOURCE_FMU_STATIC "-Wl,-Bstatic -lOpenModelicaFMIRuntimeC -lomcgc -lipopt -lcoinmumps -lcminpack -lexpat -lcdaskr -lsundials_cvode -lsundials_kinsol -lsundials_idas -lsundials_sunlinsollapackdense -lsundials_sunlinsolklu -llis -lumfpack -lklu -lamd -lbtf -lcolamd -lsuitesparseconfig -llapack -lblas -Wl,-Bdynamic -lm -ldl -lpthread -lgfortran -lstdc++ -rdynamic -Wl,--no-undefined")
+  set(RT_LDFLAGS_GENERATED_CODE_SOURCE_FMU_STATIC "-Wl,-Bstatic -lOpenModelicaFMIRuntimeC -lomcgc -lipopt -lcoinmumps -lcminpack -lexpat -lcdaskr -lsundials_cvode -lsundials_kinsol -lsundials_idas -lsundials_sunlinsollapackdense -lsundials_sunlinsolklu -llis -lumfpack -lklu -lamd -lbtf -lcolamd -lsuitesparseconfig -Wl,-Bdynamic -llapack -lblas -lm -ldl -lpthread -lgfortran -lstdc++ -rdynamic -Wl,--no-undefined")
```

Comment:
This will break some users' FMUs

Reply:
This is not final anyway. It is just a draft to see what more is missing to pass all tests. The lapack change is not actually relevant here. It was just in my commit chain.